splink/internals/predict.py

```diff
@@ -97,7 +97,9 @@ def predict_from_comparison_vectors_sqls(
         bf_terms,
         sql_infinity_expression,
     )
-
+    # Add condition to treat case of 0 as None
+    if threshold_match_probability == 0:
```

Comment:
There's whitespace at the end of these lines at the moment hence ruff is failing.

Really the best solution  is to process using ruff, but removing the whitespace manually should do it

Reply:
Hey, yeah sorry i thought I'd fixed that let me have another look

Reply:
That should have done it now, it was on line 102, sorry im very new to this stuff.